src/community/teams_responsibilities.rst

```diff
@@ -0,0 +1,39 @@
+Teams and responsibilities
+--------------------------
+
+SKA software development is organized with agile development teams.
+
+Development team
+================
+
+
+Scrum Master
+============
+The Scrum Master of each team is the responsible for the process the team
+follows. A generic description of this role can be found `on the SAFe website <https://www.scaledagileframework.com/scrum-master/>`_.
+The SKA Scrum Masters are responsible for:
```

Comment:
The list that follows is not a complete list of responsibilities...e.g. removing impediments, managing risks, protecting the team. Some of these things are described in the SAFe website. Maybe this sentence should start with "The SKA Scrum Masters are also responsible for:" to indicate that these are additional responsibilities to the generic description.